Gemfile.lock

```diff
@@ -262,6 +263,7 @@ DEPENDENCIES
   bootsnap
   cssbundling-rails
   debug
+  dotenv (~> 3.1)
```

Comment:
Learned that foreman can read .env files but needed to use dotenv in order to use the .env file in the specs